### PR TITLE
Switch Basemap to Google Maps and Cleanup the Popup

### PR DESCRIPTION
**Pull request by alfah13**
I noticed that the popup was out of date, the attributes were not descriptive enough.

## BEFORE
Screen Shot 2021-10-19 at 4 52 43 PM

## AFTER
Screen Shot 2021-10-19 at 4 52 56 PM

## BEFORE
Screen Shot 2021-10-19 at 4 51 40 PM

## AFTER
Screen Shot 2021-10-19 at 4 58 16 PM

### CODE REVIEW

**Comment by spencerbh** [comment]
Several changes made some hacky some not, furthering map functionality to render all crop parcels on map and in table.
Before https://github.com/Greenetwork/BLX_frontend_new/pull/2/commits/1fa80540ffc33449a1c04ff6bb4dd226e460de0b
previous
After https://github.com/Greenetwork/BLX_frontend_new/pull/2/commits/2751ee43464f3776a5dbc397567f5faa0e795712
post2

**Review by spencerbh — COMMENTED** [review]
new changes submitted improving and adding to your changes, 
informal review request right back at you 🪃 . 

Please drop a comment that you've pulled and tested my changes and then I'll merge

File: src/GeoJsonMap.js [hunk 1]
```diff
@@ -69,12 +72,74 @@ function Main (props) {
...
+        type: 'Feature',
+        properties: claim,
+        geometry: {
+          type: 'Polygon',
+          coordinates: [coords]
+        }
+      });
+    }
+
+    const bounds = (maxLat < -180 || maxLon < -360 || minLat > 180 || minLon > 360) ?
```
Comment (alfah13) [inline src/GeoJsonMap.js]: In WGS 84 the `LAT` range is +90 to -90 and `LON` range is +180 to -180. Where is the Lon going to 360 and LAT going to 180. (Source: https://epsg.io/4326)

File: src/GeoJsonMap.js [hunk 2]
```diff
@@ -199,38 +265,87 @@ function Main (props) {
...
                   </Table.Cell>
+                  <Table.Cell>
+                    { apnData1['crop2016'] }
+                  </Table.Cell>
+                  <Table.Cell>
+                    { apnData2['crop2016'] }
+                  </Table.Cell>
+                  <Table.Cell>
+                    { apnData3['crop2016'] }
+                  </Table.Cell>
```
Comment (alfah13) [inline src/GeoJsonMap.js:294]: What happens to the table when we have only one dict being returned for a single APN?
What happens to the table when we have over 4 dicts be ing returned for a single APN?

Comment (spencerbh) [reply src/GeoJsonMap.js:294]: 1. it has some empty columns like shown in https://user-images.githubusercontent.com/16962098/138028050-c8af2c3a-7f8b-443c-80d2-5b0fa29eafb7.png
2. It just returns the first 4. 
3. yes this is hacky, but create a dynamic table is not something i want to tackle

File: src/GeoJsonMap.js (hunk 1 above)
Comment (spencerbh) [reply src/GeoJsonMap.js]: idk

**Review by spencerbh — APPROVED** [review]

**Pull request merged**